docs/API.md

```diff
@@ -116,15 +135,15 @@ await api.getRealmBackups('1234567', '1')
 
 ### restoreRealmFromBackup
 
-(realmId: string, slotId: string, backupId: string) => Promise\<void>
+(realmId: string, slotId: string, backupId: string) => Promise\<string>
 
 Restores a Realm from a backup
 
 ```js
 await api.restoreRealmFromBackup('1234567', '1', '1970-01-01T00:00:00.000Z')
 ```
 
-No output
+Either 'Retry again later' or 'true'. Always seems to return 'Retry again later' on the first try.
```

Comment:
Shouldn't this be handled internally with an exception? This doesn't seem like good exposed API.